races.py

```diff
@@ -1,120 +1,143 @@
-        #Keys of dictionary are names of months in the form of the word the same as at the page.
-        names_of_months = { 'ledna': '01', 'února': '02', 'března': '03', 'dubna': '04', 'května': '05', 'června':'06', 'července': '07', 'srpna': '08', 'září': '09', 'října': '10', 'listopadu': '11', 'prosince': '12' }
-        #print(names_of_months)
-        name_of_month = sliced_date[2]
-        if name_of_month in names_of_months:
-            month = names_of_months[name_of_month]
-            #print(month)
-        else:
-            raise RuntimeError("U závodu není uvedeno datum v potřebném formátu.")
-
-        #Original way how I was trying to change names of months to its values.
-        #if month_in_word == 'ledna':
-            #sliced_date[2] = '01'
-        #elif month_in_word == 'února':
-            #sliced_date[2] = '02'
-        #elif month_in_word == 'března':
-            #sliced_date[2] = '03'
-        #elif month_in_word == 'dubna':
-            #sliced_date[2] = '04'
-        #elif month_in_word == 'května':
-            #sliced_date[2] = '05'
-        #elif month_in_word == 'června':
-            #sliced_date[2] = '06'
-        #elif month_in_word == 'července':
-            #sliced_date[2] = '07'
-        #elif month_in_word == 'srpna':
-            #sliced_date[2] = '08'
-        #elif month_in_word == 'září':
-            #sliced_date[2] = '09'
-        #elif month_in_word == 'října':
-            #sliced_date[2] = '10'
-        #elif month_in_word == 'listopadu':
-            #sliced_date[2] = '11'
-        #elif month_in_word == 'prosince':
-            #sliced_date[2] = '12'
-        #else:
-            #print("There is no month!")
-
-        year = sliced_date[3].replace(',', ' ').strip()
-        sliced_date[1] = year
-        sliced_date[2] = month
-        sliced_date[3] = day
-        # '-'.join(only_date) - previous used, not so understadable?
-        #only_date = sliced_date[1:4]
-        # the name of the column in our database in PostgreSQL should be the same -> "date_of_race" instead of "datum_zavodu"
-        date_of_race = year + '-' + month + '-' + day
-        print(date_of_race)
-        #f.write(date_of_race)
-
-    place = container.findAll("p", {"class" : "iframe-visible cb-iframe-place"})
-    for p in place:
-        # the name of the column in our database in PostgreSQL should be the same -> "place_of_race" instead of "misto_zavodu"
-        place_of_race = p.text.strip('()')
-        print(place_of_race)
-        #f.write(place_of_race)
-
-    name = container.findAll("h4", {"class": "mt-0 mb-0"})
-    for n in name:
-        #the name of the column in our database in PostgreSQL should be the same -> "name_of_race" instead of "nazev_zavodu"
-        name_of_race = n.text.strip()
-        print(name_of_race)
-        #f.write(name_of_race)
+page_count = 91
+date_column = []
+place_column = []
+name_column = []
+latitude = []
```

Comment:
I would suggest renaming this to `latitude_column` to keep consistency with the other variables. Same for `longitude` below.